homeassistant/components/emulated_hue/upnp.py

```diff
@@ -119,7 +107,7 @@ def run(self):
             socket.inet_aton(self.host_ip_addr))
 
         if self.upnp_bind_multicast:
-            ssdp_socket.bind(("239.255.255.250", 1900))
+            ssdp_socket.bind(("", 1900))
```

Comment:
The .net version linked is actually also stating that the port on the local machine can be anything else. Should we consider that then too ? Because odds are that we also have netdisco running on the same machine. (which btw also binds to the multicast address https://github.com/home-assistant/netdisco )